.github/workflows/tests.yml

```diff
@@ -131,7 +123,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: run-pytest
     # run both on previous step success and failure
-    if: "!cancelled()"
+    if: ${{ !cancelled() }}
```

Comment:
unrelated change: My IDE was highlighting this as errorful (I think maybe because of the double quotes?).  This is how they do it in [the docs](https://docs.github.com/en/actions/learn-github-actions/expressions#example-of-cancelled)

Reply:
Thanks! I think this is to ensure expansion/var replacement is more robust by adding the `{{ }}`